tests/Unit/Helpers/NumericalAlgorithms/FiniteDifference/Exact.hpp

```diff
@@ -0,0 +1,173 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Structure/DirectionMap.hpp"
+#include "Evolution/DgSubcell/SliceData.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeArray.hpp"
+
+namespace TestHelpers::fd::reconstruction {
+/*!
+ * \brief Test that the reconstruction procedure can exactly reconstruct
+ * polynomials of degree equal to the degree of the basis.
+ *
+ * For example, for a minmod, MC, or other second-order TVD limiter, we must be
+ * able to reconstruct linear functions exactly. For a third-order WENO scheme,
+ * we must be able to reconstruct quadratic functions exactly. This is done
+ * by setting up the function
+ *
+ * \f{align}{
+ *  u=\sum_{i=1}^p x^p + y^p + z^p
+ * \f}
+ *
+ * where \f$x,y,z\f$ are compute by the logical coordinates translated by
```

Comment:
"are compute by the logical coordinates translated" -> "are computed by translating the logical coordinates"

OR

just give the equation: "where x,y,x = xi-4,eta-4,zeta-8" or similar